### ISSUE TITLE
Running 'hence compose' without a usable file produces an odd failure message

### ISSUE THREAD

**Issue by homertherefore** [hence-io/cli#5]
After successfully connecting to a machine, running 'hence compose' without a usable file produces an odd failure message. Clearly defining the error to the user will help them ensure they use it properly, within a fold containing the right files.

``` bash
Sean@iMac ~/sites/hence $ hence compose
--------------------------------------------------------------------------------
 HENCE.io Compose
--------------------------------------------------------------------------------
No help topic for 'undefined'

  ____  _____ _   _ _____
 |  _ \|  _  | \ | | ____|
 | | | | | | |  \| |  _|
 | |_| | |_| | |\  | |___
 |____/|_____|_| \_|_____|


info: Compose `undefined` ran successfully on henceTest VM!
```

**Comment by ghost** [comment]
Should be fixed in v0.0.12.  Please try again and close on confirmation.

**Comment by homertherefore** [comment]
All clear!
